admin_interface/templates/admin/filter.html

```diff
@@ -10,13 +10,12 @@
 
     {# Use the default list filter template -> https://github.com/django/django/blob/master/django/contrib/admin/templates/admin/filter.html #}
 
-    <h3>{% blocktrans with filter_title=title %} By {{ filter_title }} {% endblocktrans %}</h3>
-    <ul>
+    <h3 {% if spec.lookup_val %}class="active"{% endif %}>{% blocktrans with filter_title=title %} By {{ filter_title }} {% endblocktrans %}</h3>
```

Comment:
`spec` is the list filter instance, `lookup_val` is the value in the query string

Reply:
Thanks for having explained it in advance.

Reply:
Should it be a comment in the code?

Reply:
I think the purpose can be easily guessed by reading the code, however a comment never hurts.

Reply:
Guessing from context is good enough, and I don’t want to have to think about whether a comment should be in which file or both so none 🙂